letter-sender/src/main/resources/application.yml

```diff
@@ -0,0 +1,16 @@
+spring:
+  cloud:
+    aws:
+      credentials:
+        access-key: ${AWS_ACCESS_KEY}
+        secret-key: ${AWS_SECRET_KEY}
+      sns:
+        endpoint: ${AWS_SNS_ENDPOINT}
+        region: ${AWS_SNS_REGION}
+
+com:
+  christmas:
+    letter:
+      aws:
+        sns:
+          topic: ${AWS_SNS_TOPIC_ARN}
```

Comment:
Does this also create the topic when it doesn't exist? Let's use the localstack-setup.sh script to create the topic.